package.json

```diff
@@ -19,13 +19,15 @@
         "test:e2e": "npx jest functional-tests.local.spec.ts",
         "lint:check": "npx lerna run lint --stream",
         "lint:fix": "npx lerna run lint:fix --stream",
-        "precheckin": "npm-run-all --serial format:check lint:check copyright:check cbuild test",
-        "format:fix": "npx prettier --write \"**/*\"",
+        "precheckin": "npm-run-all --serial format:check lint:check copyright:check syncpack:check cbuild test",
+        "format:fix": "npx syncpack format --indent \"    \" && npx prettier --write \"**/*\"",
```

Comment:
can we add [configuration file](https://github.com/JamieMason/syncpack#-configuration-file) and remove --indent as well?

Reply:
On further testing, I decided to just remove the `syncpack format` call entirely because

1. syncpack doesn't currently offer any `format:check` equivalent to enforce it in CI builds
2. none of our current `package.json` files actually respect its formatting today (it looks like the previous `cbuild`-based command didn't actually perform any of the non-indent reformatting when `fix-mismatches` and `format` were in the same command line)

We could implement our own script to do a format:check based on whether format + prettier changes anything, but if we wanted to go that route, I think it would be preferable to instead contribute upstream in https://github.com/JamieMason/syncpack/issues/3

Reply:
ok to drop, we will rely on current formatting mechanism

Reply:
Hi all, as of [12.3.0](https://github.com/JamieMason/syncpack/releases/tag/12.3.0) `syncpack lint` will now also lint formatting.